requirements/constraints.txt

```diff
@@ -15,11 +15,4 @@
 
 boto3<2.0
 
-# Not yet using Django 3.X. Using the 2.X LTS to mirror the LMS.
-django>2.2,<2.3
-
-django-cors-headers>=2.2.0,<2.3
-
-django-environ-2==2.1.0
-
-django-filter==2.1.0
+django>3.2,<3.3
```

Comment:
There don't seem to be any backwards incompatible changes relevant to this repo:

https://docs.djangoproject.com/en/3.2/releases/3.0/#backwards-incompatible-changes-in-3-0